adapter/prometheus/prometheus.go

```diff
@@ -195,6 +196,11 @@ func promValue(val adapter.Value) (float64, error) {
 		return i, nil
 	case int64:
 		return float64(i), nil
+	case time.Duration:
+		d := time.Duration(i)
+		// TODO: what is the right thing here?
+		// convert to millis for now
+		return d.Seconds() * 1e3, nil
```

Comment:
No need to create a new duration, in this `case` golang knows that `i` must be a `time.Duration`, so we can just do `return i.Nanoseconds() / int(time.Millisecond), nil`.